src/Microsoft.Sbom.Api/Workflows/Helpers/RelationshipsArrayGenerator.cs

```diff
@@ -179,4 +179,6 @@ private IEnumerator<Relationship> GetRelationships(RelationshipType relationship
             }
         }
     }
+
+    Task<GenerateResult> IJsonArrayGenerator<RelationshipsArrayGenerator>.GenerateAsync() => throw new System.NotImplementedException();
```

Comment:
Is this supposed to be here? It feels very out of place

Reply:
This seems like an outdated autogenerated stub that I had put in place for testing. I've removed it.